linters/remark-lint/plugin.yaml

```diff
@@ -0,0 +1,45 @@
+version: 0.1
+lint:
+  definitions:
+    - name: remark-lint
+      files: [markdown]
+      package: remark-cli
+      extra_packages:
+        - remark-preset-lint-consistent
+        - remark-preset-lint-recommended
+        - remark-lint-list-item-indent
+        - remark-lint-no-tabs
+        - vfile-reporter-json
+      runtime: node
+      known_good_version: 11.0.0
+      direct_configs:
+        - .remarkrc
+        - .remarkrc.json
+        - .remarkrc.cjs
+        - .remarkrc.mjs
+        - .remarkrc.js
+        - .remarkrc.yaml
+        - .remarkrc.yml
+      commands:
+        - name: format
+          output: rewrite
+          run: remark ${target} --output
+          formatter: true
+          success_codes: [0]
+          in_place: true
+          cache_results: true
+        - name: check
```

Comment:
nit: use names: `check` and `fmt`, just because that's what we tend to use for other ones

Reply:
ok